create-snowpack-app/app-template-preact-typescript/snowpack.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  mount: {
+    public: '/',
+    src: '/_dist_',
+  },
+  plugins: [
+    '@snowpack/plugin-dotenv',
```

Comment:
Maybe be we should remove @snowpack/plugin-dotenv from all templates, because it's not actually being used?

Reply:
I think we could do a better job at communicating this, but the idea is to match parity with Create React App as much as possible. But, agreed that this may be better suited as "documenting how easy it is to add this plugin" vs. "added for everyone, by default".

Lets leave it in for this PR to match all others, but great call out.